python/torch_mlir/extras/fx_importer.py

```diff
@@ -528,16 +536,16 @@ def node_val_to_type(self, node: torch_fx.Node) -> IrType:
             )
 
     def tensor_metadata_to_type(
-        self, tm: TensorMetadata, sparse_layout: torch.layout = None
+        self, tm: TensorMetadata, sparsity: Tuple[torch.layout, int, int] = None
```

Comment:
s/Tuple/tuple/

Reply:
Also per how it is used, should be `Optional[]`

Reply:
done, and done